src/components/backend-ai-resource-broker.ts

```diff
@@ -839,8 +841,14 @@ export default class BackendAiResourceBroker extends BackendAIPage {
         prefix = '';
         kernelName = specs[1];
       } else {
```

Comment:
Let's add a condition that the length of `specs` is greater than 2 for future safety: `else if (specs.length > 2)`.

Reply:
Apply it.